src/readers/morphologyHDF5.cpp

```diff
@@ -486,6 +494,29 @@ void MorphologyHDF5::_read(const std::string& groupName,
     }
 }
 
+void MorphologyHDF5::_readEndoplasmicReticulum()
+{
+    {
+        HighFive::SilenceHDF5 silence;
+
+        try {
+            const auto group = _file->getGroup(_g_endoplasmic_reticulum);
+        } catch (HighFive::GroupException&) {
```

Comment:
what's the semantics of getting this? can we check if the file has the property first, instead of catching the (potentially expensive) exception?

Reply:
This semantics comes straight from Brion. I can have a deeper look if you want.

Reply:
Nope, it's ok.

Reply:
There is a `NodeTraits::exist` method that does what you need:
https://github.com/BlueBrain/HighFive/blob/74250cf2f2a82a2b9af2f00f93475c431a2d7e09/include/highfive/bits/H5Node_traits.hpp#L110-L115

Otherwise I would consider:
```suggestion
        } catch (const HighFive::GroupException&) {
```